src/content/docs/features/state.md

```diff
@@ -34,3 +36,15 @@ export function count() {
   state.count++;
 }
 ```
+
+## What durable state unlocks
+
+Persistent `state` means you can store state in one call to a Membrane [program](/concepts/programs/) and use it again in a subsequent call. You don't need to store data in a database or file to persist it—instead, the state of your program (the JS heap) is transparently and efficiently persisted every time it changes.
+
```

Comment:
> Durability means you can treat your javascript objects as a database.

I really like "JS objects as the database", if you think it's good too, let's also drop it in the [Durable Objects](https://www.membrane.io/durable-programs) section.